src/Microsoft.Framework.PackageManager/SourceControl/ProcessUtilities.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Text;
+using Microsoft.Framework.Runtime;
+
+namespace Microsoft.Framework.PackageManager.SourceControl
+{
+    internal static class ProcessUtilities
+    {
+        public static bool ExecutableExists(string executableName)
+        {
+            string whereApp;
+            if (PlatformHelper.IsWindows)
+            {
+                whereApp = "where";
+            }
+            else
+            {
+                whereApp = "whereis";
```

Comment:
`whereis` is present on most distributions but is not part of the POSIX spec. The recommended approach is usually [`type -p`](https://en.wikipedia.org/wiki/Type_(Unix)) since that is part of the POSIX spec: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/type.html

I'm mostly OK either way for now, but it might be good to see if you can switch to that since it should be most compatible on POSIX systems.